src/components/DeploymentDetails.vue

```diff
@@ -77,6 +77,12 @@
       :alternate="alternate"
     />
 
+    <p-key-value v-if="deployment.pullSteps" label="Pull Steps" :alternate="alternate">
+      <template #value>
+        {{ deployment.pullSteps }}
```

Comment:
How does this look when its rendered? `pullSteps` is an array so I'm guessing this might just look like json?

Reply:
yeah thats correct, it looks like this

![image](https://github.com/PrefectHQ/prefect-ui-library/assets/26799928/b81c65f6-2c12-41a7-9566-6d178aafe15a)

Reply:
Could we use a p-code-highlight and show it as json? There's a `stringify` utility that will convert that array to formatted json.